src/sentry/new_migrations/monkey/writer.py

```diff
@@ -0,0 +1,37 @@
+from __future__ import absolute_import
+
+# This should be exactly the same as `django.db.migrations.writer.py.MIGRATION_TEMPLATE`,
+# except that we add
+# - `atomic = False`
+# - `is_dangerous = False`
+# to the class definition. Compare this template after each Django version bump to make
+# sure we're not missing any important changes.
+SENTRY_MIGRATION_TEMPLATE = """\
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+%(imports)s
+
+class Migration(migrations.Migration):
+    # This flag is used to mark that a migration shouldn't be automatically run in
+    # production. We set this to True for operations that we think are risky and want
+    # someone from ops to run manually and monitor.
+    # Things you should always mark as dangerous:
+    # - Adding indexes to large tables. These indexes should be created concurrently,
+    #   unfortunately we can't run migrations outside of a transaction until Django
+    #   1.10. So until then these should be run manually.
+    # - Large data migrations. Typically we want these to be run manually by ops so that
+    #   they can be monitored. Since data migrations will now hold a transaction open
+    #   this is even more important.
```

Comment:
While we're here, a good measurement I say for people, "when in doubt, just do is_dangerous".

But:

* Adding columns to highly active tables, even ones that are NULL.